src/components/Welcome/Welcome.scss

```
@@ -0,0 +1,15 @@
+.welcome {
+  height: 100vh;
+  background: coral;
+  &__text {
```

add empty line between 
```suggestion
  background: coral;
  
  &__text {
```